web/src/title/loader.ts

```diff
@@ -154,7 +154,7 @@ export function afterProjectLoaded() {
       writable(0),
     ),
   );
-  get(map)!.fitBounds(get(backend)!.getBounds(), { animate: false });
+  get(map)!.fitBounds(get(backend)!.getBounds(), { duration: 500 });
```

Comment:
And the other way: when you open a project, animate the zoom in a bit -- again, quickly.

This zoom happens after the load screen. I wanted to do this _during_ the load screen, but we'd have to mimic the backend `getBounds`, which isn't too clear to do just from the GJ savefile. We could start storing a bbox in there, but I'm not sure it's worth the hassle.

Reply:
Doing it *during* loading rather than after would be nice, but I think that could be a low priority followup task.